.github/workflows/create-release-with-version-submit.yaml

```diff
@@ -313,16 +313,27 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          RELEASE_ID=$(./scripts/create_draft_release.sh ${{ github.event.inputs.name }})
-          echo "release_id=$RELEASE_ID" >> $GITHUB_OUTPUT
+          gh release create ${{ inputs.name }} \
+            --draft \
+            --title "${{ inputs.name }}" \
+            --generate-notes \
+            --verify-tag
 
       - name: Create manifest and upload assets
         id: create-manifest
         env:
           BOT_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           IMG=$IMAGE_REPO:${{ inputs.name }} make create-manifest
-          ./scripts/upload_assets.sh ${{ steps.create-draft.outputs.release_id }}
+          MANIFEST_FILE="./manifests/btp-operator/btp-manager.yaml"
+          DEFAULT_CR_FILE="./examples/btp-operator.yaml"
+          [[ ! -e ${MANIFEST_FILE} ]] && echo "::error ::Manifest file does not exist" && exit 1
+          [[ ! -e ${DEFAULT_CR_FILE} ]] && echo "::error ::BTP operator CR does not exist" && exit 1
+          
+          gh release upload ${{ inputs.name }} \
```

Comment:
Will `gh release upload` with the tag name work? We create the tag after this step.

Reply:
Order changed in next push.